packages/adapters-library/src/adapters/ethena/tests/snapshots/ethereum.deposits.json

```diff
@@ -47,5 +47,395 @@
         "blockNumber": 19776211
       }
     ]
+  },
+  "rpc": {
+    "f13fb8b302536021b0014f6d3d1b34af": {
+      "request": {
+        "method": "eth_call",
+        "params": [
+          {
+            "to": "0xca11bde05977b3631167028862be2a173976ca11",
+            "data": "0x82ad56cb0000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000200000000000000000000000009d39a5de30e57443bff2a8307a4256c8797a349700000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000060000000000000000000000000000000000000000000000000000000000000002470a082310000000000000000000000003f843189280a4379eb12b928afd5d96df807667900000000000000000000000000000000000000000000000000000000"
+          },
+          "0x12dba9c"
+        ]
+      },
+      "response": "0x0000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000682a6ecd45edf435efda8"
+    },
```

Comment:
For the record, we do not need to store the request data, just the response. I was only storing the request for debugging purposes but now I'm wondering if we should remove it or not, as it's what allowed me to detect the unnecessary Lido calls even for non-Lido tests.

Reply:
ignore my env var suggestion, i think that would be overkill

Reply:
Let's just strip them for now.

If we need to debug something in future it's easy to temporarily modify it.